.github/workflows/reassurePerformanceTests.yml

```
@@ -0,0 +1,43 @@
+name: Reassure Performance Tests
+
+on:
+  push:
+    branches: [main]
+  pull_request:
+    branches: ['**']
+
+jobs:
+  perf-tests:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@93ea575cb5d8a053eaa0ac8fa3b40d7e05a33cc8
+
+      - name: Setup NodeJS
+        uses: Expensify/App/.github/actions/composite/setupNode@main
+
+      - name: Run performance testing script
+        shell: bash
+        run: |
+          set -e 
+          BASELINE_BRANCH=${BASELINE_BRANCH:="main"}
+          git fetch origin
+          git switch "$BASELINE_BRANCH"
+          npm install --force
+          npx reassure --baseline
+          git switch --detach -
+          npm install --force
+          npx reassure --branch
+
+      - name: Read output.md
+        id: reassure
+        uses: juliangruber/read-file-action@v1
+        with:
+          path: .reassure/output.md
+
+      - name: Add comment to PR
+        uses: actions-ecosystem/action-create-comment@cd098164398331c50e7dfdd0dfa1b564a1873fac
+        with:
+          github_token: ${{ secrets.OS_BOTIFY_TOKEN }}
+          number: ${{ github.event.pull_request.number }}
+          body: ${{ steps.reassure.outputs.content }}
```

Always leave an empty line at the end of the file
```suggestion
          body: ${{ steps.reassure.outputs.content }}
          
```